lawlibrary/views/legislation.py

```diff
@@ -10,27 +9,18 @@ def get_queryset(self):
         return super().get_queryset().filter(locality=None)
 
 
-class ProvincialLegislationView(TemplateView):
-    template_name = "lawlibrary/provincial_legislation.html"
+class LocalityLegislationView(BaseLocalityLegislationView):
     navbar_link = "legislation/provincial"
```

Comment:
All of the core logic of this view can move into `BaseLocalityLegislationView`. The only thing that really needs to live here is to customise `page_heading` to `Provincial Legislation`.